src/test/java/com/synopsys/integration/alert/performance/utility/AuditProcessingWaitJobTask.java

```diff
@@ -0,0 +1,75 @@
+package com.synopsys.integration.alert.performance.utility;
```

Comment:
We will want to add a TODO to this class to refactor to be able to remove the duplicate code in the future.  This is functional but eventually we will want to refactor this.